### DISCUSSION TITLE
App.css file size reduction option

### DISCUSSION THREAD

**Discussion by StasKoun2** [TiagoSilvaPereira/vemto-issues#166] [Ideas & Suggestions]
It would be nice to include the option to reduce the size of the app.css file (tailwind) in the project settings.
Only the classes used in the Blade, Jetstream, and Livewire templates are saved. Everything else is removed from the app.css file (tailwind)

**Comment by TiagoSilvaPereira** [comment]
Hi @StasKoun2 I moved it to the discussions, so other people can see and discuss your suggestion. Please use the Issues tab only for Bug reporting. Thanks
